### DISCUSSION TITLE
Build troubles

### DISCUSSION THREAD

**Discussion by OscarLahaie** [ocamllabs/vscode-ocaml-platform#936] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/25438086/164320170-94c856e1-67b9-44c9-8394-c6d761a09eca.png)

I don't understand why but I am not able to build.

**Comment by mlantas** [comment, marked as answer]
You need a newer version of OCaml.

The let punning feature that you're getting a syntax error on was introduced in OCaml 4.13: https://ocaml.org/releases/4.13.0.html

We use OCaml 4.14 in this project.

**Reply by OscarLahaie** [reply]
Then the makefile is outdated, no ?
